extensions/wikia/WikiFactory/Loader/WikiFactoryLoader.php

```diff
@@ -395,7 +395,7 @@ public function execute() {
 		/**
 		 * redirection to another url
 		 */
-		if( $this->mIsWikiaActive == 2 ) {
+		if( $this->mIsWikiaActive == 2 && !$this->mCommandLine ) {
```

Comment:
I would add a small comment explaining why we need it in command line mode. it may save some WTFs in the future.

Reply:
I'll add but this is happening in a couple of places in this file so somebody was taking that into account someday... but not this time.

Reply:
added in: f46127d8cc0